Formula/dtkcore@2.rb

```diff
@@ -0,0 +1,73 @@
+class DtkcoreAT2 < Formula
```

Comment:
We only use `formula@#` if there already is a formula with a different version number. That's not the case here though so you should change this to just be Dtkcore.

Reply:
I see that python@2.rb is written like this, dtkcore@2 is a released version, we will release version 5 in the near future, do I need to submit dtkcore@2.rb at that time and rename the current file to dtkcore.rb?

Reply:
Yes, the latest version is always without a version number in homebrew. (`python` is version 3 for that reason.)

Reply:
I see, I will wait until the release to submit the update, thank you.